docs/source/conf.py

```diff
@@ -89,6 +89,18 @@
 # The full version, including alpha/beta/rc tags.
 release = cartopy.__version__
 
+# Sphinx gallery configuration
+sphinx_gallery_conf = {
+    'examples_dirs': ['../../lib/cartopy/examples'],
+    'filename_pattern': '^((?!sgskip).)*$',
```

Comment:
Any file that includes `sgskip` in its title will not be run by sphinx-gallery. Right now this is only the `geostationary` example because it kept hanging on my computer and I couldn't debug it.

Reply:
So we can now remove this line, right?

Reply:
this line should stay in case you ever want to have an example that isn't run by sphinx-gallery...it's a configuration option so won't change the behavior of the docs *unless* "sgskip" is in a file name